packages/twenty-front/src/modules/billing/components/SettingsBillingSubscriptionInfo.tsx

```diff
@@ -208,6 +224,15 @@ export const SettingsBillingSubscriptionInfo = () => {
             disabled={!canSwitchSubscription}
           />
         )}
+        {isTrialPeriod && hasPermissionToEndTrialPeriod && (
+          <Button
+            Icon={IconCircleX}
+            title={t`End Trial`}
```

Comment:
End Trial feels negative, like it's going to delete my account. "Subscribe Now"?